week7.3-ex1.py

```diff
@@ -0,0 +1,198 @@
+from typing import List
+
+
+class Message:
+    """A Message class. Represents communication between people.
+
+    Parameters
+    ----------
+    sender : str
+        The sender of the message.
+    recipient : str
+        The recipient of the message.
+    title : str
+        The title of the message.
+    body : str
+        The body of the message.
+
+    Attributes
+    ----------
+     sender : str
+        The sender of the message.
+    recipient : str
+        The recipient of the message.
+    title : str
+        The title of the message.
+    body : str
+        The body of the message.
+    message_id : int
+        Incremental id of the last message sent.
+    is_read : bool
+        If the message has been read already.
+    """
+    def __init__(self, sender: str, recipient: str, title: str, body: str):
+        self.sender = sender
+        self.recipient = recipient
+        self.title = title
+        self.body = body
+        self.is_read = False
+        self.message_id = 1
+
+    def add_message_id(self, message_id: int):
```

Comment:
Why would we change the message id and if so why 'add_message' and not 'change_message'.

Reply:
Right. This is a change of the number because the constructor initializes it to 1.